tools/export/makefile/__init__.py

```diff
@@ -200,7 +200,7 @@ class GccArm(Makefile):
 
     @staticmethod
     def prepare_lib(libname):
-        return "-l:" + libname
+        return "-l" + libname[3:-2]
```

Comment:
It _should_ be, but you can easily generate a library by using the ar command yourself. I think there are versions of mbed OS 5 and 2 that contain libraries that do not conform. It should be pretty safe to add that check.